### DISCUSSION TITLE
Rename `change` Parameter to `e` in OnPropertyChanged

### DISCUSSION THREAD

**Discussion by robloo** [AvaloniaUI/Avalonia#9779] [General]
The event args parameter always seemed incorrectly named to me here:

```csharp
        /// <summary>
        /// Called when a avalonia property changes on the object.
        /// </summary>
        /// <param name="change">The property change details.</param>
        protected virtual void OnPropertyChanged(AvaloniaPropertyChangedEventArgs change)
        {
        }
```

I would expect the name `e` (most common for other methods) or even `args` instead of `change`.

Has anyone else noticed this? Is it worth changing for 11.0 before everything is locked down? (It's a pretty big change)

@grokys

**Comment by kekekeks** [comment]
I'm pretty sure that changing parameter names doesn't affect compatibility at all.

**Reply by robloo** [reply]
Yes, sorry, I shouldn't have said "breaking change". It is still a big change though so want to be sure it's directionally correct. The inheritdoc functionality also breaks if the parameter names don't match.
